src/routes/Root/Root.tsx

```diff
@@ -45,36 +30,14 @@ const Root = () => {
           }}
         />
       </header>
-      <header className="bg-jod-primary p-3">
-        <h1>{t('header')}</h1>
-        <ul className="flex flex-wrap gap-3">
-          {['en', 'fi', 'sv'].map((lng) => (
-            <li key={lng}>
-              <NavLink to={`/${lng}`} className="block p-3">
-                {lng}
-              </NavLink>
-            </li>
-          ))}
-        </ul>
-        <ul className="flex flex-wrap gap-3">
-          {links.map((link) => (
-            <li key={link.text}>
-              <NavLink to={link.to} className="block p-3">
-                {link.text}
-              </NavLink>
-            </li>
-          ))}
-        </ul>
-      </header>
-      <main className="p-3">
+      <main>
         <Routes>
           <Route index element={<Home />} />
           <Route path={`${t('slugs.profile.index')}/*`} element={<Profile />} />
           <Route path={`${t('slugs.about-us')}/*`} element={<AboutUs />} />
           <Route path="*" element={<NoMatch />} />
         </Routes>
       </main>
-      <footer className="bg-jod-secondary p-3">{t('footer')}</footer>
```

Comment:
As mentioned in other comment, designs for the footer is not "finished" so removed it for now.